### PR TITLE
fix: wrong change index + add slug

### PR DESCRIPTION
**Pull request by vdegenne**

### CODE REVIEW

**Review by VandeurenGlenn — CHANGES_REQUESTED** [review]

File: packages/api/src/cache.ts [hunk 1]
```diff
@@ -51,15 +60,11 @@ export const init = async () => {
...
 
-	try {
-		// Making sure cache directory exists
-		await mkdir(CACHE_ROOT_DIRECTORY)
-	} catch {
-		// Ignore if it does
-	}
+	await createCacheDirectory()
 
 	writeFile(join(CACHE_ROOT_DIRECTORY, 'cache.json'), JSON.stringify(cache.bluelist))
```
[the remark this answers is not in the document]
Comment (vdegenne) [reply packages/api/src/cache.ts:69]: `createCacheDirectory` does not produce any error so it's fine.

[the remark this answers is not in the document]
Comment (VandeurenGlenn) [reply packages/api/src/cache.ts:69]: Yes but it runs always, even when not needed, if writeFails we know there is no cachedir

[the remark this answers is not in the document]
Comment (vdegenne) [reply packages/api/src/cache.ts:69]: What's your suggestion?

[the remark this answers is not in the document]
Comment (vdegenne) [reply packages/api/src/cache.ts:69]: Maybe just accept and make the change as you see, would be faster

**Review by VandeurenGlenn — APPROVED** [review]

**Pull request merged**